tests/jobs/rft/test_gendata_rft.py

```diff
@@ -117,12 +137,32 @@ def test_gendata_rft_csv(tmpdir, input_data):
     assert (~dframe[~dframe["is_active"]]["inactive_info"].isnull()).all()
 
 
-def test_gendata_rft_directory(tmpdir, input_data):
+def test_gendata_rft_csv_reek(tmpdir, reek_data):
```

Comment:
Good with an integration test, but would like to see an integration test of `update_simdata_from_rft` as well.

Reply:
Wouldn't that be a unit test? I skipped it because there were none such already. Is there enough difference between the integration test and such a unit test to warrant having that extra test layer?

Reply:
I meant unit test, was just about to correct myself now ;) I also noticed there was no unit test, and would be good to have one, as it makes debugging easier.

Reply:
Added in `test_trajectory_rftfile.py`.